.github/workflows/build.yml

```diff
@@ -369,15 +366,48 @@ jobs:
           artifact_name: ${{ env.PKG_NAME }}_${{ needs.get-product-version.outputs.product-version }}_${{ matrix.goos }}_${{ matrix.goarch }}.zip
         run: |
           echo "::group::Unpack and Prep"
-          docker pull registry.access.redhat.com/ubi7/ubi-minimal:7.9-1057
           unzip "$artifact_name"
           echo "::group::Diagnostics"
           echo "CGO related build information:"
           go version -m ./nomad | grep CGO
           echo "GLIBC links:"
           go tool nm ./nomad | grep -i glibc | cut -d @ -f 2-3 | sort --version-sort | uniq
           echo "::group::Smoke test binary"
-          docker run --rm -v "$PWD:/src" registry.access.redhat.com/ubi7/ubi-minimal:7.9-1057 /src/nomad version
+          docker run --rm -v "$PWD:/src" redhat/ubi8:8.10 /src/nomad version
+
+  minimum-os-arm64:
+    # test glibc 2.28 compatibility (RHEL 8.10)
+    name: OS Compatibility ARM64
+    needs:
+      - get-go-version
+      - get-product-version
+      - build-linux
+    runs-on: ubuntu-22.04-arm
+    strategy:
+      fail-fast: false
+      matrix:
+        goos: [linux]
+        goarch: [arm64]
```

Comment:
Perhaps dumb question: can the `runs-on` be interpolated via the matrix, so that we can have one `minimum-os` job that gets interpolated into the various architectures?

Reply:
yeah... here's what I'm trying:
```
        goarch: [amd64, arm64]
        runner: [ubuntu-22.04, ubuntu-22.04-arm]
        exclude:
          - runner: ubuntu-22.04
            goarch: arm64
          - runner: ubuntu-22.04-arm
            goarch: amd64
    runs-on: ${{ matrix.runner }}
```
I'm gonna see how this runs now. Matrices are not ideal here because we don't want a 2-dimensional or 1-dimensional matrix here, what we want is two pairs. But let's see. At least it avoids the copy-pasta nature of having two steps with identical contents.